cmd/terramate/cli/cloud_sync_terraform_plan.go

```diff
@@ -33,10 +33,13 @@ func (c *cli) getTerraformChangeset(run stackCloudRun, planfile string) (*cloud.
 		return nil, errors.E(clitest.ErrCloudInvalidTerraformPlanFilePath, "path must be relative to the running stack")
 	}
 
-	absPlanFilePath := filepath.Join(run.Stack.HostDir(c.cfg()), planfile)
-	_, err := os.Lstat(absPlanFilePath)
-	if err != nil {
-		return nil, errors.E(err, "checking plan file")
+	// Terragrunt writes the plan to a temporary directory, so we cannot check for its existence.
+	if !run.Task.UseTerragrunt {
+		absPlanFilePath := filepath.Join(run.Stack.HostDir(c.cfg()), planfile)
+		_, err := os.Lstat(absPlanFilePath)
+		if err != nil {
+			return nil, errors.E(err, "checking plan file")
+		}
```

Comment:
What about removing this altogether? I feel like it could create more issues in the future. Wdyt?
I don't have strong opinions, so feel free to address this or not.

Reply:
I'll leave it in for now, because this path returns an error, and the other checks don't, but print a warning instead. This is probably fine, but I'd rather not change existing behaviour without testing it some more.